src/com/tw/go/plugin/nuget/NuGetFeedDocument.java

```diff
@@ -69,7 +75,7 @@ public PackageRevision getPackageRevision(boolean lastVersionKnown) {
 
     private String getReleaseNotes() {
         String releaseNotes = getProperty(getProperties(), "ReleaseNotes");
-        if (releaseNotes == null || releaseNotes.trim().isEmpty()) return null;
```

Comment:
There is no occurrence of `getReleaseNotes()` & `getProjectUrl()` in this PR. can you tell me where is the test for this.